src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/MetaType.php

```diff
@@ -101,8 +104,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'choice_translation_domain' => false,
             ])
             ->add('page_title', TranslatableType::class, [
+                'type' => TextWithRecommendedLengthType::class,
                 'required' => false,
                 'options' => [
+                    'recommended_length' => SeoSettings::RECOMMENDED_TITLE_LENGTH,
```

Comment:
Don't you forget 'required: false' in options array ?

Reply:
Yes you're right 😑this PR was screwed because I attempted to fuse 2 previous PRs in one and the git merge did badly...

Reply:
FYI, if you set nothing it's false by default https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Form/Admin/Type/TranslatableType.php#L106